test/instrumentatios/static-instrumentations/grpc-js/generated_proto/hello_pb.d.ts

```diff
@@ -0,0 +1,69 @@
+// package: helloworld
+// file: hello.proto
+
+/* tslint:disable */
+/* eslint-disable */
+
+import * as jspb from 'google-protobuf';
```

Comment:
same -> consider limiting the import if possible

Reply:
same, generated